src/views/pacbio/PacbioRunIndex.vue

```diff
@@ -171,8 +199,14 @@ export default {
     redirectToRun(runId) {
       this.$router.push({ path: `/pacbio/run/${runId || 'new'}` })
     },
-
     ...mapActions('traction/pacbio/runs', ['fetchPacbioRuns', 'updateRun']),
+    ...mapActions('traction/pacbio/runCreate', ['fetchSmrtLinkVersions']),
+    async provider() {
```

Comment:
Nice. I think this logic should live in the DataFetcher and it gets passed the fetchers object, but for now this is good as otherwise we would have to change all the other places that DataFetcher is used, maybe add a TODO or something.

I also noticed that the FilterCard uses this function so it will refetch the SMRTLink versions each time which isn't necessary but its not a big issue and i'm not sure if there is a nice way to avoid it.